packages/Rakefile

```diff
@@ -73,6 +79,8 @@ class GroongaPackageTask < PackagesGroongaOrgPackageTask
     case key
     when "APACHE_ARROW_VERSION"
       detect_target_apache_arrow_version
+    when "APACHE_ARROW_NEXT_VERSION"
```

Comment:
How about `APACHE_ARROW_NEXT_MAJOR_VERSION`?
Because the next version may not a major release.
For example, the next version of 6.0.0 is 6.0.1.

```ruby
  def detect_apache_arrow_next_version
    detect_target_apache_arrow_version.split(".", 2)[0].succ
  end
```

```spec
 BuildRequires:	arrow-devel = @APACHE_ARROW_VERSION@, arrow-devel < @APACHE_ARROW_NEXT_VERSION@.0.0
```